docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: '3'
+services:
+  db:
+    image: postgres
+    volumes:
+      - ./tmp/db:/var/lib/postgresql/data
```

Comment:
I don't know if there's a way to specify this volume at runtime or set it to a more generic path, but we don't want production data to be keep in a `tmp` directory. The best place would probably be something like `~/crimethinc/data` where assuming crimethinc has its own user on the host machine.